ports/stm32f4/boards/STM32F401_fs.ld

```diff
@@ -5,10 +5,10 @@
 /* Specify the memory areas */
 MEMORY
 {
-    FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 512K /* entire flash */
+    FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 384K /* entire flash */
     FLASH_ISR (rx)  : ORIGIN = 0x08000000, LENGTH = 16K /* sector 0 */
     FLASH_FS (rx)   : ORIGIN = 0x08004000, LENGTH = 48K /* sectors 1,2,3 are 16K */
-    FLASH_TEXT (rx) : ORIGIN = 0x08010000, LENGTH = 448K /* sector 4 is 64K, sectors 5,6,7 are 128K */
+    FLASH_TEXT (rx) : ORIGIN = 0x08010000, LENGTH = 320K /* sector 4 is 64K, sectors 5,6,7 are 128K */
```

Comment:
Why change this? It doesn't look like the Pico uses it but something else may.

Reply:
I just mixed up my linker files like a chucklehead, thanks for catching it

Reply:
renamed them so they're more clear too